tests/DoctrineTestSuite1.php

```diff
@@ -70,7 +71,8 @@ public static function suite()
         $suite->addTestSuite('WriteAPIsiteMethodsTests');
         $suite->addTestSuite('WriteAPIserviceMethodsTests');
         $suite->addTestSuite('WriteAPIendpointMethodsTests');
-        $suite->addTestSuite('org\gocdb\tests\ManageAPICredentialsTest');
+        $suite->addTestSuite('org\gocdb\tests\ManageUnusedAPICredentialsTest');
+        $suite->addTestSuite('org\gocdb\tests\ManageUnrenewedAPICredentialsTest');
```

Comment:
these seem over indented compared to the line above, could you update?

Reply:
Not on my view? 
![image](https://user-images.githubusercontent.com/20706670/228585611-983c63a4-6d8d-423d-89d2-5c0960096ccb.png)

Reply:
I also see them as being correctly indented